### PR TITLE
Remove unused authentication service injection in main.component.ts file

### LINKED ISSUE #1177: Remove unused authentication service injection in main.component.ts file
> Remove unused authentication service injection in main.component.ts file
>     public authService: AuthenticationService

From #1151

### PR DESCRIPTION
**Pull request by eddiejaoude**
closes #1177

### CODE REVIEW

**Comment by eddiejaoude** [comment]
AuthService is used in `html`...

```html
<div class="mat-subheading-2">{{ authService.isAuthenticated ? authService.profile.name : 'Guest' }}</div>
```

Errors...

```
ERROR in src/app/dialog/list/dialog-list.component.html(8,27): : Property 'toDate' does not exist on type 'Date'.
src/app/main.component.html(4,18): : Property 'authService' does not exist on type 'MainComponent'.
src/app/main.component.html(5,23): : Property 'authService' does not exist on type 'MainComponent'.
src/app/main.component.html(6,43): : Property 'authService' does not exist on type 'MainComponent'.
src/app/main.component.html(6,43): : Property 'authService' does not exist on type 'MainComponent'.
src/app/main.component.html(7,38): : Property 'authService' does not exist on type 'MainComponent'.
src/app/main.component.html(7,38): : Property 'authService' does not exist on type 'MainComponent'.
src/app/main.component.html(20,27): : Property 'authService' does not exist on type 'MainComponent'.
src/app/main.component.html(38,76): : Property 'authService' does not exist on type 'MainComponent'.
src/app/main.component.html(39,77): : Property 'authService' does not exist on type 'MainComponent'.
src/app/main.component.html(50,19): : Property 'authService' does not exist on type 'MainComponent'.
src/app/main.component.html(51,19): : Property 'authService' does not exist on type 'MainComponent'.
src/app/main.component.html(52,86): : Property 'authService' does not exist on type 'MainComponent'.
src/app/main.component.html(53,103): : Property 'authService' does not exist on type 'MainComponent'.
src/app/projects/repository/repository.component.html(14,28): : Property 'events' does not exist on type 'RepositoryModel'.
src/app/projects/repository/repository.component.html(30,28): : Property 'releases' does not exist on type 'RepositoryModel'.
```

File: web/src/models/event.model.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import { RepositoryModel } from './repository.model';
+
+export class EventModel {
+    uid: string;
+    type: string;
+    public: string;
+    actor: UserModel;
+    repository: RepositoryModel;
+    // organisation: input.org ? GitHubOrganisationMapper.import(input.org) : {},
+    // payload: GitHubPayloadMapper.import(input.type, input.payload),
```
Comment (eddiejaoude) [inline web/src/models/event.model.ts]: This will be implemented in a future issue

**Comment by eddiejaoude** [comment]
CI all green now 😎

File: web/src/app/main.component.ts [hunk 2]
```diff
@@ -60,11 +61,27 @@ export class MainComponent {
     ];
 
     constructor(
-        public authService: AuthenticationService,
+        private authService: AuthenticationService,
```
Comment (brettminnie) [inline web/src/app/main.component.ts]: Question here, why is the object being passed to the constructor private? Is this some strange javascript voodoo :D

Comment (eddiejaoude) [reply web/src/app/main.component.ts]: Yep 😂. In typescript and angular if `private` in the ts component then it won't be available to the corresponding `html`

**Comment by brettminnie** [comment]
Interesting, not used to declaring visibility of arguments in real language
:laughing:, thanks for clarifying

**Comment by eddiejaoude** [comment]
> Interesting, not used to declaring visibility of arguments in real language 😆, thanks for clarifying
> […](#)

No worries, thanks for the question 🤓

**Pull request merged**